nexus/src/app/background/sync_switch_port_settings.rs

```diff
@@ -0,0 +1,1240 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Background task for propagating per-port switch settings to management daemons
```

Comment:
I have already begun breaking some of the steps down into functions due to similar thoughts, the primary driver being that I want other to be able to read the code in `activate()` and be able to understand what it is doing at a high level. The bootstore and BGP stuff hasn't been broken down yet because it's going through another round of refactoring (primarily the bootstore update logic, which depends on the BGP update logic).